pkg/rancher-desktop/components/TheTitle.vue

```diff
@@ -31,6 +28,23 @@ export default Vue.extend({
         this.isChild = current.path.lastIndexOf('/') > 0;
       },
     },
+    action: {
+      handler(current) {
+        if (!current) {
+          return;
+        }
+
+        if (componentCache[current]) {
+          this.dynamicComponent = componentCache[current];
+        } else {
+          import(`@pkg/components/${ current }.vue`).then((component) => {
```

Comment:
Nobody looks at the return value for this function, and we don't _need_ it happen synchronously, right?

I think that means this can be simplified to:
```ts
      async handler(current) {
        if (current) {
          componentCache[current] ||= (await import(`@pkg/components/${ current }.vue`)).default;
          this.dynamicComponent = componentCache[current];
        }
      },
```

Reply:
I quite like this implementation, I think we should use it 🙂